.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -44,69 +48,151 @@ export interface Props {
   metadata?: Metadata;
 }
 
+/*
+ * Clickable card: link properties
+ * 1) Href and text are required attributes
+ * 2) Text will be used by screen readers; it will only be visible when the
+ * link is focused by a keyboard
+ * 3) Target attribute is optional
+ */
+export type LinkType = {
+  href: string /* 1 */;
+  text: string /* 2 */;
```

Comment:
I'd like this to be a `React.Node` because if the link does open in a new tab, devs might want to add screen reader-only text like "opens in a new tab" and an icon or something like that. (Could also just put it in parenthesis but I feel like that's weird for sighted keyboard users.)